task/common/machine/script.go

```diff
@@ -32,6 +32,15 @@ base64 --decode << END | sudo tee /usr/bin/tpi-task > /dev/null
 END
 chmod u=rwx,g=rx,a=rx /usr/bin/tpi-task
 
+sudo tee /usr/bin/tpi-task-shutdown << 'END'
+#!/bin/bash
+if [[ "${CI}" ]]; then
```

Comment:
I like my conditionals to be a bit more explicit, but it's a matter of taste. Anyway; wouldn't you prefer to remove those extra curly braces?

```suggestion
if ! test -z "$CI"; then
```

```suggestion
if [[ "$CI" ]]; then
```

Reply:
> Anyway; wouldn't you prefer to remove those extra curly braces

For me is a template no brainer. I use it when I have to expand or not